bin/ci/ubuntu/install-dependencies.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash -u
-# Exit if anything fails.
-set -e
+# Exit if anything fails. Echo commands to aid debugging.
+set -ex
+
+# Target working dir - defaults to current dir.
+# Can be set from caller, or in the first parameter
+TWD=$( cd ${TWD:=${1:-${PWD:-`pwd`}}}; pwd )
```

Comment:
Nit: It's a little strange to use both `$()` and `\`\`` on the same line. I prefer `$()`.